Assets/RTLTMPro/Scripts/Editor/ContextMenu.cs

```diff
@@ -1,4 +1,5 @@
-﻿using TMPro;
+﻿using Codice.CM.Client.Differences;
```

Comment:
Is this namespace needed?

Reply:
This is most likely have happened due to auto-complete. I'll remove that asap.